.github/workflows/ci-run-test.yml

```diff
@@ -389,12 +389,25 @@ jobs:
           path: |
             coverage/*
             !coverage/.gitkeep
+      - name: Debug branch name
```

Comment:
I'll leave this in until we have it sorted on `master` 😬 Once it works, we can remove it.